_posts/0000-01-02-cesar-nogueira.md

```diff
@@ -0,0 +1,6 @@
+---
+ layout : título do slide
+ : " Bem-vindo ao nosso segundo slide! "
+---
+Seu texto 
```

Comment:
É muito fácil colocar algumas palavras em ** negrito ** e outras palavras em * itálico * com Markdown. Você pode até mesmo [link para o Google!] (Http://google.com)